datafusion/sql/src/select.rs

```diff
@@ -555,3 +565,288 @@ fn match_window_definitions(
     }
     Ok(())
 }
+
+fn create_function_physical_name(
```

Comment:
Does this functionality need to remain in sync with the creation of physical names?